mailing-list-api/server.js

```diff
@@ -0,0 +1,60 @@
+import express from "express";
+import mailinglist from "./mailing-lists.js";
+const PORT = 3000;
+
+const app = express();
+app.use(express.json());
+app.disable("x-powered-by");
```

Comment:
Is there any particulaur reason for this line of code? It's a good question to ask yourself whenever you find anything odd in code.